python/ray/rllib/utils/filter_manager.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+
+
+class FilterManager(object):
+    """Manages filters and coordination across remote evaluators that expose
+        `get_filters` and `sync_filters`.
+    """
+
+    def __init__(self, **filters):
+        self._filters = filters
+
+    def synchronize(self, remotes):
```

Comment:
Hm it's kind of weird that you take filters as a class argument, and remotes as a method argument. Is there value in doing that?

Consider making FilterManager a pure function that takes both in as arguments.